<?xml version="1.0"?>
<launch>

    <!-- launch-prefix="xterm -e gdb -ex run args" -->

    <node pkg="ros_examples" type="example_rrt_exploration_ogm_with_filtering_and_assigner" name="example_rrt_exploration_ogm_with_filtering_and_assigner" output="screen" respawn="true">
        <param name="debug" value="false"/>
        <!-- eta must be greater than map resolution -->
        <param name="eta" value="0.5"/>
        <param name="min_dist" value="0.25"/>
        <param name="marker_size" value="0.025"/>
        <param name="rate_hz" value="1000"/>
        <param name="map_topic" value="/map"/>
        <!-- <param name="map_topic" value="/projected_map"/> -->
        <remap from="clicked_point" to="/clicked_point"/>
        <remap from="rrt" to="rrt"/>
        <remap from="frontier" to="frontier"/>
    </node>

    <include file="$(find ros_examples)/launch/simulator.launch">
        <arg name="rviz_file" default="example_rrt_exploration_ogm_with_filtering_and_assigner.rviz"/>
    </include>

    <include file="$(find ros_examples)/launch/navigation_stack.launch" />

    <!-- <node pkg="rviz" type="rviz" name="rviz" args="-d $(find ros_examples)/rviz/example_rrt_exploration_ogm_with_filtering_and_assigner.rviz"/>
    <node pkg="rosbag" type="play" name="play" args="clock loop /home/office/2020-11-09-20-16-14.bag"/> -->


</launch>